app/_universe/[world_id]/page.tsx

```
@@ -16,9 +16,9 @@ import {
 } from "remeda";
 import { Tables } from "@/database.types";
 import { FloatingSessionViewer } from "@/components/FloatingSessionViewer";
-import { assert } from "console";
 import { notFound } from "next/navigation";
 import { PageBase } from "@/components/PageBase";
+import assert from "assert";
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)